src/main/java/com/g4s8/ghman/bot/Notification.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.bot;
+
+import com.g4s8.ghman.user.Thread;
+import com.g4s8.ghman.user.Users;
+import com.g4s8.teletakes.bot.BotSimple;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.telegram.telegrambots.api.methods.send.SendMessage;
+import org.telegram.telegrambots.exceptions.TelegramApiException;
+
+/**
+ * Notify user into telegram chat.
+ * @since 1.0
+ */
+public interface Notification {
+
+    /**
+     * Sends notification.
+     * @throws IOException When fails
+     */
+    void send() throws IOException;
+
+    /**
+     * Unread threads notification.
+     * @since 1.0
+     * @todo #9:30 Continue implement notifications for user when new messages
+     *  (unread github notification) arrive. Unread msgs can be retrieved with
+     *  the help of PgThreads#unread() method, and this class is meant to sent
+     *  them to tlg chat. So set up whole mechanism together.
```

Comment:
@olenagerasimova I'm not clear what is the next step here:
- what is meant to be using `Notification`?
- where is `UnreadThreads` meant to be instantiated?

Can you elaborate a bit better on what you had in mind when you introduced those abstractions?

Reply:
@victornoel `Notification` mission is simple - send message (in our case - to telegram chat), and `UnreadThreads` implementation does that - it sends telegram messages to all given users about all given github threads. 

So, now it's necessary to monitor unread notification by periodically checking `threads` table for unread threads. Saying more practical, the whole idea is to schedule java thread (or any other synchronization process) to periodically call `PgThreads#unread()` to retrieve unread messages and `Notification.UnreadThreads` to send them. 

Do you get the idea now? Or smth is still not clear?

Reply:
@olenagerasimova I do understand now, could you reword the todo to explain it?